{{cookiecutter.repostory_name}}/dc-prod.yml

```diff
@@ -1,26 +1,28 @@
-version: '2'
+version: '3.7'
 
 services:
   redis:
     image: redis:3-alpine
+    init: true
     command: redis-server --appendonly yes
     volumes:
       - ./redis/data:/data
 
   db:
     image: postgres:9.6-alpine
+    init: true
     environment:
       - POSTGRES_DB=${POSTGRES_DB}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
     volumes:
       - ./db/data:/var/lib/postgresql/data
 
-
   app:
     build:
       context: ./app
       dockerfile: Dockerfile
     image: {{cookiecutter.django_project_name}}/app
+    init: true
```

Comment:
I know init is necessary in `celery-worker` and `app`, but is it needed in others?

Reply:
From what I know `pg`, `redis` don't spawn any additional subprocesses and aren't started from shell so zombies shouldn't leak but being extra-cautious with `init: true` doesn't cost much, at least I don't see any slow down

Reply:
it costs a little bit of memory and if pg or redis would create zombies, it would be a bug in their apps and they'd have to fix it. Lets not waste memory on an init process that will never do anything.

Reply:
Removed `init: true` from `redis`, `postgres` and `nginx`. I left it in all celery-related services as they import our django code.